DocsReadMe.md

```diff
@@ -130,6 +130,59 @@ bool isConverted = invalidSecondValue.TryToAnother(out AnotherEnum anotherEnum);
 // [isConverted, anotherEnum] - [false, First]
 ```
 
+# ToAnotherOrDefault
+
+**Description**
+
+Convert input enum to another enum or default value if not possible.
```

Comment:
Hmm, lets add more details
input enum -> `enumValue`
another enum -> `TAnotherEnum`
default value -> `defaultValue`

Reply:
Then it would be different from other descriptions in DocsReadMe file. Should I change them too?

Reply:
Oh well, you're right... 
Sorry, let's revert this changes and will follow single style